pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -255,11 +261,27 @@ func (r *Reconciler) reconcileDeployment(logger *zap.SugaredLogger, el *v1alpha1
 		return err
 	}
 
-	labels := mergeMaps(el.Labels, GenerateResourceLabels(el.Name))
+	//labels := mergeMaps(el.Labels, GenerateResourceLabels(el.Name))
```

Comment:
was this intentional?

Reply:
looks like we call mergeMaps in line 330. So, we can remove this line?

Reply:
Oh forgot to remove commented line

Done